### PR TITLE
Fix memory collect logs

### PR DESCRIPTION
**Pull request by arxcruz**
According ansible documentation, we should avoid use sudo when use fetch
module because it uses slurp to make the checksum and might consume all
the memory

### CODE REVIEW

File: playbooks/collect-logs.yml [hunk 1]
```diff
@@ -158,11 +158,13 @@
       fetch: src=/tmp/{{ inventory_hostname }}.tar.gz flat=yes dest={{ base_dir }}/{{ inventory_hostname }}.tar.gz validate_checksum=no
       ignore_errors: true
       when: job.gzip_logs is not defined
+      sudo: no
```
Comment (tkammer) [inline playbooks/collect-logs.yml]: please use "become: no" since "sudo" is deprecated

File: playbooks/collect-logs.yml [hunk 2]
```diff
@@ -158,11 +158,13 @@
...
 
     - name: fetch log archive (tar)
       fetch: src=/tmp/{{ inventory_hostname }}.tar flat=yes dest={{ base_dir }}/{{ inventory_hostname }}.tar validate_checksum=no
       ignore_errors: true
       when: job.gzip_logs is defined and job.gzip_logs
+      sudo: no
```
Comment (tkammer) [inline playbooks/collect-logs.yml]: same as above

**Comment by rhosqeauto** [comment]
ltgm, no need to sudo/become anyone, as these files should be readable anyway (mabe ensure that? chmod +r whatever.tar.[g/x/b]z )

**Comment by queria** [comment]
ltgm, no need to sudo/become anyone, as these files should be readable anyway (mabe ensure that? chmod +r whatever.tar.[g/x/b]z )

**Pull request closed without merging**